### DISCUSSION TITLE
My users can't connect using walletconnect / issue since yesterday

### DISCUSSION THREAD

**Discussion by GoldenTakeishi** [WalletConnect/walletconnect-monorepo#2884] [Developer Support] [answered]
Hello, so my website https://presale.goldeninutoken.org look to not allow anyone to connect using the QR code or even trustwallet in the browser tab nothing happen...

I don't understand where my issue is coming from, i have implented the V2 since V1 got shutdown and still getting trouble... Can someone help me find a solution to it.

**Comment by finessevanes** [comment]
Hey - sorry you're having issues. Could we get the following to help isolate the issue?
1) Version of the package you are using
2) Reproducible code via a codeBox, error logs, and/or reproduction steps
3) Which wallet (platform and version) you used to get to the issue
4) Do you get the issues on https://react-web3wallet.vercel.app/

**Reply by Valentin529** [reply]
it seems that viem sends request to https://rpc.walletconnect.com/v1/?chainId=eip155:97 and server response 500 error.
Having same issue 
"@web3modal/ethereum": "^2.6.2",
"@web3modal/html": "^2.6.2",
"ethers": "^6.6.1",
 "viem": "^1.2.6",
 Neither wallet or public provider works

**Reply by Valentin529** [reply]
<img width="354" alt="Screenshot 2023-06-30 at 13 02 26" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/68601123/b7763b74-17c4-487a-817a-3b54fa4fc5ac">

**Reply by EthanBlockson** [reply, marked as answer]
RPC was down https://status.walletconnect.com/

**Reply by Valentin529** [reply]
Thanks, issue fixed
